### DISCUSSION TITLE
Show more checks (jobs) on pull request page at one time

### DISCUSSION THREAD

**Discussion by flaksp** [community/community#17402] [Pull Requests]
Currently, only 6 first checks displayed at pull request page, and if there's more, scrollbar will appear:

<img width="1145" alt="Screenshot 2022-05-27 at 22 20 23" src="https://user-images.githubusercontent.com/12474739/170777294-76889115-4744-49a9-8cb2-c5d0df09b883.png">

I noticed that scrolling this tiny block annoys me a bit each time I start or review PRs. It feels like it's too small for me. I believe this limit may be safely increased from 6 to 10-20 checks.

**Comment by bjornpijnacker** [comment]
GitHub doesn't seem keen to change this soon.
Using an extension such as Stylus may help. The following CSS will show all checks:
```
.merge-status-list {
    max-height: initial !important;
}
```
